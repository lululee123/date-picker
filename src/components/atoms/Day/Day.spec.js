import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';

import RouteSection from './index';

it('renders RouteSection correctly', () => {
	const component = renderer
		.create(
			<BrowserRouter>
				<RouteSection />
			</BrowserRouter>,
		)
		.toJSON();
	expect(component).toMatchSnapshot();
});
