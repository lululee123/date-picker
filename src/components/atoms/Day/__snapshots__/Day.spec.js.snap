// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Day correctly 1`] = `
<button
  className="day normal"
  onClick={[Function]}
  type="button"
>
  
</button>
`;
